src/index.ts

```diff
@@ -17,17 +15,17 @@ const renderUsers = async (config: ApiConfig) => {
   });
 };
 
-const app = () => {
-  const config = (window as any).__CONFIG__;
-  delete (window as any).__CONFIG__;
-
```

Comment:
You can register the config right here, so you don't need to export it into the ioc file.

Reply:
thanks, moved from ioc and register config there